### PR TITLE
Added missing "@charset "UTF-8";" from style.css

### PR DESCRIPTION
**Pull request by Swapnildhanrale**
Fixes https://github.com/WordPress/twentytwenty/issues/748

### CODE REVIEW

**Comment by nielslange** [comment]
@Swapnildhanrale Thanks for your PR. Can you fix the Travis issue by adding `/* stylelint-disable-line */` behind `@charset "UTF-8";` and also update the `style-rtl.css` accordingly?

Also, @maheshwaghmare might have been interested in fixing this issue on his own. While it's great that you fixed it instead, it's also nice to give reported the option to create a PR on the issues they report. 😉

File: style.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@charset "UTF-8";
+@charset "UTF-8"; /* stylelint-disable-line */
```
Comment (nielslange) [inline style.css]: @ntwb Can Would you mind taking a look why Travis still shows 

>  2:1  ✖  Expected empty line before comment   comment-empty-line-before

in https://travis-ci.com/WordPress/twentytwenty/jobs/241941980 despite `/* stylelint-disable-line */` had been added behind `@charset "UTF-8";`? Should the statement look like this instead:

```
/* stylelint-disable-line custom-property-empty-line-before */
```

Comment (Swapnildhanrale) [reply style.css]: Yes checking

Comment (nielslange) [reply style.css]: @Swapnildhanrale I think @ntwb needs to take a look here. 😉 While you could add an empty row before the statement to solve this issue, the file should not start with an empty line.

**Comment by acosmin** [comment]
@Swapnildhanrale Put a space between the charset declaration and the beginning of the next comment block

```
@charset "UTF-8";

/*
Theme Name: Twenty Twenty
```

**Comment by nielslange** [comment]
> @Swapnildhanrale Put a new line between the charset declaration and the beginning of the next comment block
> 
> ```
> @charset "UTF-8";
> 
> /*
> Theme Name: Twenty Twenty
> ```

@acosmin Isn't it the other way around? In https://travis-ci.com/WordPress/twentytwenty/jobs/241941980 Travis complains shows

> style.css
> 2:1  ✖  Expected empty line before comment   comment-empty-line-before

Thus, instead of

```
@charset "UTF-8";

/*
Theme Name: Twenty Twenty
```

I think the fix would be

```

@charset "UTF-8";
/*
Theme Name: Twenty Twenty
```

Having said that, starting the document with an empty line sounds pretty wrong to me. That's why I asked @ntwb for some assistance on how to bypass the Stylelint issue.

**Comment by nielslange** [comment]
> > @Swapnildhanrale Put a new line between the charset declaration and the beginning of the next comment block
> > ```
> > @charset "UTF-8";
> > 
> > /*
> > Theme Name: Twenty Twenty
> > ```
> 
> @acosmin Isn't it the other way around? In https://travis-ci.com/WordPress/twentytwenty/jobs/241941980 Travis complains shows
> 
> > style.css
> > 2:1  ✖  Expected empty line before comment   comment-empty-line-before
> 
> Thus, instead of
> 
> ```
> @charset "UTF-8";
> 
> /*
> Theme Name: Twenty Twenty
> ```
> 
> I think the fix would be
> 
> ```
> 
> @charset "UTF-8";
> /*
> Theme Name: Twenty Twenty
> ```
> 
> Having said that, starting the document with an empty line sounds pretty wrong to me. That's why I asked @ntwb for some assistance on how to bypass the Stylelint issue.

@Swapnildhanrale Welcome to Open Source! 😂

**Comment by acosmin** [comment]
It says there line 2 `2:1` 😄  On line two there's the `/*` beginning... When I run lint on my end there isn't any issues with my solution.

**Comment by nielslange** [comment]
> It says there line 2 `2:1` 😄 On line two there's the `/*` beginning... When I run lint on my end there isn't any issues with my solution.

@acosmin: 🤦‍♂️ Question to myself: _"What is a comment?!?"_ You are absolutely right! The lint issue does not refer to `@charset "UTF-8";` but to the comment and the comment starts in line 2 but should start in line 3. Thanks for helping me realise this. I've totally missed that. 😜

@Swapnildhanrale Can you forget about everything I said so far, accept my apologies, remove `/* stylelint-disable-line custom-property-empty-line-before */` and add an empty line between `@charset "UTF-8";` and the comment. This should solve the Travis issue.

**Comment by ianbelanger79** [comment]
This is unnecessary so I am closing.

Closes #748

**Pull request closed without merging**